protobufs/py/setup.cfg

```diff
@@ -0,0 +1,25 @@
+[metadata]
+name = p4runtime
```

Comment:
There's also same directory structure under `networking-recipe/stratum/p4runtime/py/` (which is part of the ipdk-io/p4runtime repo).

Can we not reuse/update that instead of duplicating this effort and maintaing two different sources of code?

Reply:
I suppose I could copy the collateral from `p4runtime` instead of maintaining local copies, or at least some of it.

I suspect we're going to want to supply our own modified versions of the config files, if for no other reason than to have the metadata make it clear that this is a non-canonical p4runtime wheel. Also, the new minimum versions you propose are based on the ones we , not the minimum levels that the p4runtime project assumes.

I had also thought of providing our own README.md file that explained the nature of the wheel, and then linked to the original README on GitHub.

That leaves `setup.py` and `LICENSE` to be copied from `p4runtime`.